source/contribute/maintainer.rst

```diff
@@ -10,129 +10,53 @@ access to the main repository of a project. In the example of the ORM,
-default it'll track your origin remote's master:
-
-.. code-block:: console
-
-    $ git config branch.master.remote doctrine
-
-Optionally, add any additional contributor/maintainer forks, e.g.:
-
-.. code-block:: console
-
-    $ git remote add romanb git://github.com/romanb/orm.git
+-  Tagging released versions within release branches.
 
 Branching Model
 ---------------
 
 Merging topic branches:
 
--  Topic branches **must** merge into **master** and/or any affected
-   release branches.
--  Merging a topic branch puts it into the *next* release, that is the
-   next release created from **master** and/or the next patch release
-   created from a specific release branch.
-
-Configuring Remotes
--------------------
-
-Add remote repo for contributor/maintainer, if necessary (only needs to
-be done once per maintainer):
-
-.. code-block:: console
-
-    $ git remote add hobodave git://github.com/hobodave/orm.git
-
-Fetch remote:
-
-.. code-block:: console
-
-    $ git fetch hobodave
-
-Merge topic branch into master:
-
-.. code-block:: console
-
-    $ git checkout master
-    Switched to branch 'master'
-    $ git merge --no-ff hobodave/DDC-588
-    Updating ea1b82a..05e9557
-    (Summary of changes)
-    $ git push doctrine master
-
-The **--no-ff** flag causes the merge to always create a new commit
-object, even if the merge could be performed with a fast-forward. This
-avoids losing information about the historical existence of a topic
-branch and groups together all commits that together added the topic.
+- Topic branches **must** merge into the lowest supported branch when
+  they are a bugfix, or an improvement that does not affect stability
+  (documentation, CI pipeline, tests).
+- Topic branches **must** merge into the next minor release if they
+  are a new feature that does not involve a backwards-compatibility
+  break, or a deprecation.
+- Topic branches **must** merge into the next major release if they
+  contain a backwards-compatibility break. Reviewers and contributors
+  should try hard to think of a way to make it backwards compatible and
+  contribute it to the next minor instead. Ideally, the next major
+  release should be about removing deprecated features.
 
 Release Branches
 ----------------
 
--  May branch off from: **master**
--  Must merge back into: **master**
--  Branch naming convention: 1.0, 2.0, 2.1
+- May branch off from the next minor or major branch.
+- Branch naming convention: 1.0.x, 2.0.x, 2.1.x
 
-Release branches are created when **master** has reached the state of
-the next major or minor release. They allow for continuous bug fixes and
-patch releases of that particular release until the release is no longer
-supported.
+Release branches are created either manually or through
+``laminas/automatic-releases`` (depending on the configuration) when the
+next minor or major branch has reached the state of the next major or
+minor release, so that contributors can submit new features.
```

Comment:
I don't understand that sentence. It sounds like an "or" or an "and" is missing.

Reply:
I also think that this sentence should be split into two or more sentences to make it easier to understand.

Reply:
I don't think any word is missing but it's indeed hard to read. I'll rework it.